.travis.yml

```diff
@@ -7,32 +7,32 @@ cache:
         - vendor
         - $HOME/.composer/cache/files
 
-php:
-    - 5.6
-    - 7.0
-    - 7.1
-    - hhvm
-
-env:
-    - TWIG_EXT=no
-    - TWIG_EXT=yes
+matrix:
+    include:
+        - php: hhvm-stable
+          sudo: required
+          dist: trusty
+          group: edge
+          env: TWIG_EXT=no
+        - php: 7.0
+          env: TWIG_EXT=no
+        - php: 7.1
+          env: TWIG_EXT=no
+    allow_failures:
+        - php: hhvm-stable
+          sudo: required
+          dist: trusty
+          group: edge
+          env: TWIG_EXT=no
```

Comment:
I think you can simplify this ``allow_failures`` group to ``php: hhvm-stable`` only